src/pages/achievement/dashboard/subcomponents/utils/AchievementDeadline.tsx

```diff
@@ -0,0 +1,37 @@
+import { Icon } from '@blueprintjs/core';
+import { IconNames } from '@blueprintjs/icons';
+import React from 'react';
+
+import { AchievementAbility } from '../../../../../features/achievement/AchievementTypes';
+import { prettifyDeadline } from './DateHelper';
+
+type AchievementDeadlineProps = {
+  deadline?: Date;
+  ability: AchievementAbility;
+};
+
+function AchievementDeadline(props: AchievementDeadlineProps) {
+  const { deadline, ability } = props;
+
+  const oneDay = 86400000;
```

Comment:
Is there a javascript way to manipulate days without using this constant.

Reply:
Yep I have fixed it thanks!